src/cljfx/fx/node.clj

```diff
@@ -24,6 +27,13 @@
     :disable [:setter lifecycle/scalar :default false]
     :effect [:setter lifecycle/dynamic]
     :event-dispatcher [:setter lifecycle/scalar]
+    ; takes an event description that receives filtered Events with event type Event/ANY
+    :event-filter (prop/make
+                    (mutator/adder-remover
+                      #(.addEventFilter ^Node %1 Event/ANY ^EventHandler %2)
+                      #(.removeEventFilter ^Node %1 Event/ANY ^EventHandler %2))
+                    lifecycle/event-handler
+                    :coerce coerce/event-handler)
```

Comment:
Sorry for nitpicking, but I want a minor correction here: `composite/props` macro transforms vector values to `prop/make` calls to reduce boilerplate, and thus preferred over `prop/make` calls:
```clj
[(mutator/added-remover ...)
 lifecycle/event-handler
 :coerce coerce/event-handler]
```